inductiva/resources/machines_base.py

```diff
@@ -104,6 +106,11 @@ def _register_machine_group(self, **kwargs):
     def __repr__(self):
         pass
 
+    def current_machines_to_str(self) -> str:
+        """Return the number of machines currently running in the MPI cluster.
```

Comment:
Is this comment for MPI?

Reply:
No, copy pasted and forgot to change the docs.